src/Frontend/Page/Admin.hs

```diff
@@ -44,6 +45,8 @@ data PageAdminSettingsQuorum =
 
 instance Page PageAdminSettingsQuorum
 
+-- FIXME: the following names are a little ridiculous.  s/PageAdminSettingsGaPUsersView/GroupPermUserView/?9
```

Comment:
> We could drop the `PageZZZ` preffixes from the other page types as well. Wdyt?

ok, fine with me.  i'm actually in favour of adding them to all page types consistently. but i don't think we have to vote on that, since i already know @np's opinion on this.  :)